pyanaconda/storage_utils.py

```diff
@@ -489,3 +489,8 @@ def reset_to_snapshot(self, storage, dispose=False):
 # a snapshot of early storage as we got it from scanning disks without doing any
 # changes
 on_disk_storage = StorageSnapshot()
+
+def filter_unsupported_disklabel_devices(devices):
```

Comment:
This is a terrible name for a function, but I don't have a better one. Except for maybe ``get_suported_disklabel_devices()``? It's at least a bit shorter.